docs/components/power-sensor/_index.md

```diff
@@ -0,0 +1,239 @@
+
+Return the current of a specified device and whether it is AC or DC.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` [(Optional[Dict[str,Any]])](https://docs.python.org/library/typing.html#typing.Optional): Pass additional data and configuration options to the [RPC call](/appendix/glossary/#term-grpc).
+- `timeout` [(Optional[float])](https://docs.python.org/3/library/typing.html#typing.Optional): Specify a time limit (in seconds) for how long `get_current` should wait for a response.
+
+**Returns:**
+
+- [(Tuple[float, bool])](https://docs.python.org/3/library/functions.html#float): A float representing the current reading in amps. A bool indicating whether the current is AC (True) or DC (False).
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/power_sensor/power_sensor/index.html#viam.components.power_sensor.power_sensor.PowerSensor.get_current).
+
+```python
+# Get the current reading from the power sensor
+current, is_ac = await my_power_sensor.get_current()
+print("The current is ", current, " A, Is AC: ", is_ac)
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [Context](https://pkg.go.dev/context): Control the lifecycle of the operation by handling timeouts and managing cancellations.
+- `extra`[(Optional[Dict[str, Any]])](https://docs.python.org/3/library/typing.html#typing.Optional): Pass additional data and configuration options to the [RPC call](/appendix/glossary/#term-grpc).
+
+**Returns:**
+
+- `float64`: The measurement of the current, represented as a 64-bit float number.
+- `bool`: Indicate whether current is AC (True) or DC (False).
+- `error`: Report any errors that might occur during operation. Successful operation will be `nil`.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/powersensor#PowerSensor).
+
+```go
+// Create a power sensor instance
+myPowerSensor, err := powersensor.FromRobot(robot, "my_power_sensor")
+if err != nil {
+  logger.Fatalf("cannot get power sensor: %v", err)
+}
+
+// Get the current reading from device in amps
+current, isAC, err := myPowerSensor.Current(context.Background(), nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetVoltage
+
+Return the voltage reading of a specified device and whether it is AC or DC.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra`[(Optional[Dict[str, Any]])](https://docs.python.org/3/library/typing.html#typing.Optional): Pass additional data and configuration options to the [RPC call](/appendix/glossary/#term-grpc).
+- `timeout`[(Optional[float])](https://docs.python.org/3/library/functions.html#float): Specify a time limit (in seconds) for how long `get_voltage` should wait for a response.
+
+**Returns:**
+
+- [(Tuple[float, bool])](https://docs.python.org/3/library/stdtypes.html): A float representing the current reading in amps. A bool indicating whether the voltage is AC (True) or DC (False).
```

Comment:
nit: in addition to `true` / `false`, let's specify that we return a tuple, formed of a float and a bool.:

```suggestion
- [(Tuple[float, bool])](https://docs.python.org/3/library/stdtypes.html): A tuple which includes a float representing the current reading in amps, and a bool indicating whether the voltage is AC (`true`) or DC (`false`).
```